src/ftxui/dom/separator.cpp

```diff
@@ -52,11 +59,21 @@ class SeparatorWithPixel : public Separator {
 };
 
 Element separator() {
-  return std::make_shared<Separator>();
+  return std::make_shared<Separator>(LIGHT);
 }
 
-Element separator(Pixel pixel) {
-  return std::make_shared<SeparatorWithPixel>(pixel);
```

Comment:
If the removal of the `Element separator(Pixel);` overload was intentional its forward declaration should've also been removed from `elements.hpp`.